bin/travis

```diff
@@ -3,9 +3,9 @@
 set -e
 
 if [[ $TRAVIS_SCALA_VERSION == 2.11* ]] && [[ $SCALAZ_VERSION = 7.2* ]]; then
-  SBT_COMMAND=";coverage ;clean ;test ;makeSite ;coverageOff"
+  SBT_COMMAND="\"set scalazVersion in ThisBuild := \\\"7.2.1\\\"\" ;coverage ;clean ;test ;makeSite ;coverageOff"
```

Comment:
I am a little unsure about this because testing it locally, the semicolons seem to screw things up so I am not sure how this works on Travis.

But assuming it currently works, I think this SHOULD work. I guess we'll find out by inspecting the Travis output.